x-pack/plugins/fleet/server/services/agents/status.ts

```diff
@@ -83,7 +103,11 @@ export async function getAgentStatusForAgentPolicy(
     }
   );
 
-  return {
+  if (pitId) {
+    await closePointInTime(esClient, pitId);
+  }
+
+  const result = {
     total: allActive.total,
```

Comment:
Inactive agents are the ones that have been unenrolled, so they're generally excluded from the status as they're not relevant most of the time. We should probably have two fields, one for the real total and one for the total active, but I think we should only do this in a non-breaking way. Maybe `total` (all active, deprecated) and `all` (all active and inactive) and `active` (all active)

Reply:
I agree, this would be cleaner.
Do you happen to know if there is any usage of `other` status? It might be good to deprecate that as well, as it seems confusing. It is not mentioned in the UI/docs either.

Reply:
I agree it is confusing. Total seems to be total active, which you don't assume from the name total, my vote would be to fix this.

Reply:
Raised an issue to deprecate total status: https://github.com/elastic/kibana/issues/135980